app/models/location.rb

```diff
@@ -1,4 +1,9 @@
 class Location < G5Updatable::Location
   has_many :phone_numbers
   has_many :ppc_numbers
+
+  def country_code_prefix
+    country_code = properties['country_code'] ? properties['country_code'] : "1"
```

Comment:
Added some defense here, so once this code gets pushed live, and prior to running `heroku run rake g5_updatable:load_all -a g5-phone-number-service` the API will return `country_code : "+1-"` for all locations.  This way the endpoint doesn't have a bunch of `country_code : "+-"` values prior to the rake task coming through.  Probably doesn't really matter to have a value of `+-` in the endpoint until the code in the phone-service.js in the CMS is updated to look at the `country_code` key in the endpoint.  Overkill?